kv/document.go

```diff
@@ -81,6 +81,12 @@ func (s *Service) FindDocumentStore(ctx context.Context, ns string) (influxdb.Do
 	})
 
 	if err != nil {
+		if IsNotFound(err) {
```

Comment:
I don't think this solves #12864. This function is just used to lookup a namespace not the actual document.

Reply:
Thanks a lot